projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -156,22 +157,35 @@ export class DotFormComponent {
         })
             .then(async (response: Response) => {
                 if (response.status !== 200) {
+                    console.log('----handleSubmit 400');
                     const error: DotHttpErrorResponse = {
                         message: await response.text(),
                         status: response.status
                     };
                     throw error;
                 }
+                console.log('----handleSubmit 200');
                 return response.text();
             })
             .then((_text: string) => {
-                // Go to success page
+                const formReturnUrl = this.getFormReturnUrl();
+                if (formReturnUrl) {
+                    window.location.href = formReturnUrl;
+                }
             })
             .catch(({ message, status }: DotHttpErrorResponse) => {
+                console.log('----catch', message);
                 this.errorMessage = message || fallbackErrorMessages[status];
             });
     }
 
+    private getFormReturnUrl(): string {
+        const fieldFormReturn = getFieldsFromLayout(this.layout).filter(
+            (field: DotCMSContentTypeField) => field.variable === 'formReturnPage'
+        )[0];
+        return fieldFormReturn.values;
+    }
+
```

Comment:
maybe this code should go into a util class, and 'formReturnPage' should be a read only attribute

Reply:
@freddyucv  IMO, not needed since this is only used on the dot-form and it's only for the success submit condition